src/FSharp.Build/FSharp.Build.fsproj

```diff
@@ -67,6 +67,12 @@
       <Pattern1>{{FSCorePackageVersionValue}}</Pattern1>
       <Replacement1>$(FSCorePackageVersionValue)-$(VersionSuffix)</Replacement1>
     </NoneSubstituteText>
+    <NoneSubstituteText Include="Microsoft.FSharp.Core.NetSdk.props" CopyToOutputDirectory="PreserveNewest" Condition="'$(Configuration)' != 'Proto'">
+      <TargetFileName>Microsoft.FSharp.Core.NetSdk.props</TargetFileName>
+      <SubDir>PreRelease\</SubDir>
+      <Pattern1>{{FSCorePackageVersionValue}}</Pattern1>
+      <Replacement1>$(FSCorePackageVersionValue)-$(_PreReleaseLabel).final</Replacement1>
```

Comment:
This uses slightly different creation  logic than `FSharpCorePreviewPackageVersionValue` . Is that correct?